eo-maven-plugin/src/test/java/org/eolang/maven/MjProbeTest.java

```diff
@@ -9,72 +9,20 @@
 import com.yegor256.WeAreOnline;
 import java.io.IOException;
 import java.nio.file.Path;
-import org.cactoos.io.InputOf;
 import org.cactoos.io.ResourceOf;
-import org.eolang.jucs.ClasspathSource;
-import org.eolang.parser.EoSyntax;
-import org.eolang.xax.XtSticky;
-import org.eolang.xax.XtYaml;
-import org.eolang.xax.XtoryMatcher;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.junit.jupiter.params.ParameterizedTest;
 
 /**
  * Test case for {@link MjProbe}.
  *
  * @since 0.28.11
- * @todo #4203:90min Make {@link MjProbeTest} To Check Probes
- *  Currently, this test case only checks the number of probes found
- *  but does not verify the actual content of the probes.
- *  It would be beneficial to enhance this test to
- *  check the actual probes found.
- * @todo #4203:90min Enable {@link MjProbeTest#findsProbesInSimpleProgram(Path)} Test.
- *  This test is currently disabled because it fails.
- *  We should investigate the cause of the failure and enable the test.
- *  Originally this issue affected the integration tests,
- *  <a href="https://github.com/objectionary/eo/issues/4203">here</a>
  */
 @ExtendWith(WeAreOnline.class)
 @ExtendWith(MktmpResolver.class)
 final class MjProbeTest {
-    @Test
-    void findsProbes(@Mktmp final Path temp) throws Exception {
-        final String expected = "11";
-        MatcherAssert.assertThat(
-            String.format(
-                "Number of objects that we should find during the probing phase should be equal %s",
-                expected
-            ),
-            new FakeMaven(temp)
-                .with("foreignFormat", "json")
-                .withProgram(MjProbeTest.program())
-                .execute(new FakeMaven.Probe())
-                .programTojo()
-                .probed(),
-            Matchers.equalTo(expected)
-        );
-    }
-
-    @ParameterizedTest
-    @ClasspathSource(value = "org/eolang/maven/probe-packs/", glob = "**.yaml")
-    void checksProbePacks(final String yaml) {
```

Comment:
@volodya-lombrozo looks like this test can stay in the `MjProbeTest`, since we don't need `Probes` in this test. WDYT?